src/AdminConsole/TagHelpers/LocalTime.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Passwordless.AdminConsole.TagHelpers;
+
+[HtmlTargetElement("asp-local-time")]
+public class LocalTime : TagHelper
+{
+    [HtmlAttributeName("datetime")]
+    public DateTime DateTime { get; set; }
+
+    public override void Process(TagHelperContext context, TagHelperOutput output)
+    {
+        output.TagName = "local-time";
+        output.Attributes.Add("datetime", DateTime.SpecifyKind(DateTime, DateTimeKind.Utc).ToString("O"));
```

Comment:
Should probably be `ToString("O", CultureInfo.InvariantCulture)`

Reply:
If it's ISO 8601 it likely wouldn't matter.